### DISCUSSION TITLE
Start Menu does not stay open

### DISCUSSION THREAD

**Discussion by ruebennase56** [Open-Shell/Open-Shell-Menu#797] [Q&A]
Hello Everyone,

Since the last Microsoft Win10 update (KB5004237, 21H1) the Start Menu pops up for a tenth of a second and closes again.
Has anyone a idea how to fix this, looks like a settings problem to me. I'm using Open-Shell-Menu v4.4.169. 

Thanks
Andreas

**Comment by nuclearkacey** [comment]
I have the same problem!

**Comment by ruebennase56** [comment]
Here is a workaround.

https://github.com/Open-Shell/Open-Shell-Menu/issues/795#issuecomment-892940811

**Reply by mmalloy0** [reply]
This worked and crazy how anyone figured this out, but THANK YOU!
